.github/workflows/release.yaml

```diff
@@ -0,0 +1,61 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+      - ci/publish-pipeline
+
+permissions: {}
+jobs:
+  release:
+    permissions:
+      contents: write # to push chart release and create a release (helm/chart-releaser-action)
+      packages: write # needed for ghcr access
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3.5
+        with:
+          version: v3.12.0
+
+      - name: Add dependency chart repos
+        run: |
+          helm repo add truecharts https://charts.truecharts.org
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.5.0
+        with:
+          charts_dir: ./
+          config: ./.github/ci/cr.yaml
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_SKIP_EXISTING: "false"

```

Comment:
What if we set this to true?

Reply:
You are right, we left it false just to make a test. Gonna replace it